tests/test_berry.py

```diff
@@ -67,24 +72,40 @@ def compare_quant(quant):
         return quant
 
 
-
-def test_Fe(check_integrate, system_Fe_W90, compare_energyresult,quantities_Fe,Efermi_Fe):
+def test_Fe(check_integrate,system_Fe_W90, compare_energyresult,quantities_Fe,Efermi_Fe):
```

Comment:
why removing spaces ?

Reply:
I don't know why. There is a  conflict after merge this line.